mne/minimum_norm/tests/test_inverse.py

```diff
@@ -322,10 +322,10 @@ def test_localization_bias():
     fwd = convert_forward_solution(fwd, force_fixed=True, surf_ori=True)
     fwd = fwd['sol']['data']
     want = np.arange(fwd.shape[1])
-    for method, lower, upper in (('MNE', 35, 40),
-                                 ('dSPM', 45, 50),
-                                 ('sLORETA', 55, 60),
-                                 ('eLORETA', 50, 55)):
+    for method, lower, upper in (('MNE', 83, 87),
+                                 ('dSPM', 96, 98),
+                                 ('sLORETA', 100, 100),
+                                 ('eLORETA', 100, 100)):
```

Comment:
@agramfort looks like the reason these were so low before was because the `fwd` was being converted with `use_cps=False` and the inverse implicitly being created with `use_cps=True`. Now that this PR sets them both to `True` by default, these numbers are a lot higher, which is nice.